TransferMethod/Sources/SelectTransferMethodTypePresenter.swift

```diff
@@ -302,9 +304,12 @@ final class SelectTransferMethodTypePresenter {
         view.transferMethodTypeTableViewReloadData()
     }
 
-    private func trackUILoadImpression() {
+    func trackUILoadImpression() {
+        guard loadedSuccessfully else {
```

Comment:
Can you please explain the scenario when `trackUILoadImpression()` will be called and `loadedSuccessfully` will be false?

Reply:
yes, after the creation of VC from the method `viewDidAppear`

Reply:
I tested locally, from `viewDidAppear` `loadedSuccessfully` is already true

Reply:
fixed